File: widget/src/main/java/com/yzq/widget/drop_down_menu/DeviceUtils.java
package com.yzq.widget.drop_down_menu;

import android.content.Context;
import android.graphics.Point;
import android.view.WindowManager;

/**
 * 设备工具箱，提供与设备硬件相关的工具方法
 */
public class DeviceUtils {

    /**
     * 获取屏幕尺寸
     */

    public static int getScreenHeight(Context context) {
        WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);

        if (wm == null) {
            return -1;
        }
        Point point = new Point();
        wm.getDefaultDisplay().getRealSize(point);
        return point.y;

    }


}

File: widget/src/main/java/com/yzq/widget/drop_down_menu/DeviceUtils.kt
package com.yzq.widget.drop_down_menu

import android.content.Context
import android.graphics.Point
import android.view.WindowManager
import com.blankj.utilcode.util.Utils


object DeviceUtils {
    /**
     * 获取屏幕尺寸
     */
    @JvmStatic
    fun getScreenHeight(context: Context): Int {

        val wm = Utils.getApp().getSystemService(Context.WINDOW_SERVICE) as WindowManager
        val point = Point()
        wm.defaultDisplay.getRealSize(point)
        return point.y

    }
}